org.uqbar.project.wollok.lib/src/wollok/lang.wlk

```diff
@@ -1281,7 +1281,11 @@ class String {
 	 * 		"become".contains("CO")      ==> Answers false
 	 */
 	method contains(other) {
-		return self.indexOf(other) > 0
+		try {
+			return self.indexOf(other) >= 0
+		} catch e {
```

Comment:
:+1: We shouldn't  catch everything here. 

Also, the expression in the return can never return false, since now indexOf always returns   a number `>= 0`.

We could improve this code by either: 
   * preserving an internal version of `indexOf` with the original behaviour
   * by not using indexOf in the implementation of contains
   * by introducing a new, generic message like `{ self.indexOf(other)  }.throws(NoSuchElementException)`  which better reveals the intention of the code